src/engraving/CMakeLists.txt

```diff
@@ -23,6 +23,8 @@ set(MODULE_QRC engraving.qrc)
 
 include(${CMAKE_CURRENT_LIST_DIR}/libmscore/libmscore.cmake)
 include(${CMAKE_CURRENT_LIST_DIR}/compat/midi/compatmidi.cmake)
+include(${CMAKE_CURRENT_LIST_DIR}/layout/v0/layout.cmake)
+include(${CMAKE_CURRENT_LIST_DIR}/layout/v20230717/layout.cmake)
```

Comment:
Maybe we should just call them stable/unstable? seems to me that these numbers will confuse developers a lot. And also, we probably only need two versions of the layout: main and under development

Reply:
Renamed